astro.config.ts

```diff
@@ -36,6 +37,13 @@ export default defineConfig({
 			changefreq: ChangeFreq.DAILY,
 			priority: 0.7,
 			lastmod: new Date(),
+			i18n: {
+				defaultLocale: "en",
+				locales: Object.keys(languages).reduce((prev, key) => {
+					prev[key] = key;
```

Comment:
It isn't currently a concern (as there are no locales in `translations.json` that do this), but we may want to put these values through [`fileToOpenGraphConverter()`](https://github.com/unicorn-utterances/unicorn-utterances/blob/f2ce0dfb33b7ba66de3573f8742eb7a6d1b480d3/src/utils/translations.ts#L10) in `utils/translations` to convert from the `en-us` value (as currently described in [CONTRIBUTING.md](https://github.com/unicorn-utterances/unicorn-utterances/blob/main/CONTRIBUTING.md#finding-a-language-code)) to `en_US` as in ISO 639-1

(likewise for the meta tags)